dlt/pipeline/pipeline.py

```diff
@@ -519,6 +519,7 @@ def load(
         *,
         workers: int = 20,
         raise_on_failed_jobs: bool = False,
+        truncate_staging: bool = True,
```

Comment:
Motivation to use an argument in `run()` and `load()` is that every request to cloud databases costs money. User may be willing to avoid frequent truncation.